pkg/client/cli/cmds_intercept.go

```diff
@@ -294,11 +294,12 @@ func newInterceptState(
 	}
 }
 
-func interceptMessage(r *connector.InterceptResult) string {
+func interceptMessage(r *connector.InterceptResult) error {
 	msg := ""
+	cat := errcat.Unknown
```

Comment:
Sort of a nit so take it or leave it but... This is how my mind works, but I think I'd prefer a var name like "category" or "errCategory" or "errCat" just because when I see "cat" I first think of the `cat` command, followed by a pet, and I don't think I'll see this and think it's short for category.